src/models/simple_metabolism_model.py

```diff
@@ -135,22 +130,21 @@ def get_iob_from_sbr(self, sbr_actual):
         num_hours_post_t0 = 8
```

Comment:
instead of fixing this to 8 hours would default this to the insulin action duration

Reply:
I purposely refactored to the point of being one step away from being easily configurable but not actually configured yet to get input from you and Jason. So I'll go ahead and require a DIA for the model and set the value there.